docs/ref/models/constraints.txt

```diff
@@ -284,11 +284,14 @@ in the ``ordering`` column.
 The error code used when ``ValidationError`` is raised during
 :ref:`model validation <validating-objects>`. Defaults to ``None``.
```

Comment:
I think we can make these docs updates clearer 🙂 

Firstly, I think we should have a `versionchanged` rather than `versionadded` note and be aware that if someone doesn't read the note, they shouldn't lose any information on how this works.

We have information of the default behaviour saying here `Defaults to None` and below `Defaults to
:attr:.BaseConstraint.violation_error_message.`. This is not true in the case that `:attr:~UniqueConstraint.fields` is used without a `:attr:~UniqueConstraint.condition` and I think this should be spelled out that there are 2 default cases. 
If `unique_together` does get deprecated (as the docs mention it might in future), then this default behaviour would get simplified but currently there are two default messages.

Reply:
> Firstly, I think we should have a `versionchanged` rather than `versionadded` note and be aware that if someone doesn't read the note, they shouldn't lose any information on how this works.

This is done thank you 👍 

> We have information of the default behaviour saying here `Defaults to None` and below `Defaults to :attr:.BaseConstraint.violation_error_message.`. This is not true in the case that `:attr:~UniqueConstraint.fields` is used without a `:attr:~UniqueConstraint.condition` and I think this should be spelled out that there are 2 default cases. If `unique_together` does get deprecated (as the docs mention it might in future), then this default behaviour would get simplified but currently there are two default messages.

This is not done, let me try to explain.
```
:ref:`model validation <validating-objects>`. Defaults to ``None``.
```
There is no reference for the case this is **not true** and explaining when the error code is the same as when using `Field.unique` or `Meta.unique_together`.

We shouldn't have information explaining how it works contained in the versionchanged note that doesn't exist in the definition (as these notes get deleted in time).

Same comment for `.. attribute:: UniqueConstraint.violation_error_message` 👍